docs/book/driver-deployment/upgrade.md

```diff
@@ -4,14 +4,82 @@
+    ```
+
+4. unmount and remount volume at the same location with the same mount options used for mounting volume. For this step, the `nfs-common` package needs to be pre-installed on the worker VMs.
+
+   1. Umount the volume
+      - Use the `umount -f` command to unmount the volume with force.
+
+         ```bash
+         root@k8s-worker3:~# umount -f /var/lib/kubelet/pods/43686ba4-d765-4378-807a-74049fca39ee/volumes/kubernetes.io~csi/pvc-7e43d1d3-2675-438d-958d-41315f97f42e/mount
+         ```
+
+      - If the volume is busy or already inaccessible, use `umount -l` command to do the lazy unmount.
+
+         ```bash
+         root@k8s-worker3:~# umount -l /var/lib/kubelet/pods/43686ba4-d765-4378-807a-74049fca39ee/volumes/kubernetes.io~csi/pvc-7e43d1d3-2675-438d-958d-41315f97f42e/mount
+         ```
+
+   2. Remount the volume with the same mount options used originally.
+
+        ```bash
+         root@k8s-worker3:~# mount -t nfs4 -o sec=sys,minorversion=1  10.83.28.38:/52d7e15c-d282-3bae-f64d-8851ad9d352c /var/lib/kubelet/pods/43686ba4-d765-4378-807a-74049fca39ee/volumes/kubernetes.io~csi/pvc-7e43d1d3-2675-438d-958d-41315f97f42e/mount
+        ```
+
+5. confirm mount point is accessible from the Node VM
+
+    ```bash
+    root@k8s-worker3:~# mount | grep pvc-7e43d1d3-2675-438d-958d-41315f97f42e
+    10.83.28.38:/52d7e15c-d282-3bae-f64d-8851ad9d352c on /var/lib/kubelet/pods/43686ba4-d765-4378-807a-74049fca39ee/volumes/kubernetes.io~csi/pvc-7e43d1d3-2675-438d-958d-41315f97f42e/mount type nfs4 (rw,relatime,vers=4.1,rsize=1048576,wsize=1048576,namlen=255,hard,proto=tcp,timeo=600,retrans=2,sec=sys,clientaddr=10.83.26.244,local_lock=none,addr=10.83.28.38)
+
+    root@k8s-worker3:~# ls -la /var/lib/kubelet/pods/43686ba4-d765-4378-807a-74049fca39ee/volumes/kubernetes.io~csi/pvc-7e43d1d3-2675-438d-958d-41315f97f42e/mount
+    total 4
+    drwxrwxrwx 3 root root    0 Aug  9 16:48 .
+    drwxr-x--- 3 root root 4096 Aug  9 18:40 ..
+    -rw-r--r-- 1 root root    6 Aug  9 16:48 test
+    ```
+
+After all vSAN file share volumes are remounted on the worker VMs, you can proceed with upgrading the driver.
+
+We are in the process of automating the above steps so until we have automation available we have to execute the workaround step mentioned above manually.
+
+## Upgrade vSphere CSI Driver
+
 To upgrade from one release to another release of vSphere CSI driver, we recommend to completely delete the driver using the original YAML files, you have used to deploy the driver and then use new YAML files to re-install the vSphere CSI driver.
 
-For example, if you want to upgrade driver from `v2.0.0` release to `v2.0.1` release deployed on the Kubernetes Cluster installed on `vSphere 67u3` release
+To upgrade driver from `v2.0.0` release to `v2.0.1` release deployed on the Kubernetes Cluster installed on `vSphere 67u3` release
 
```

Comment:
Can we mention that this is an example of upgrading driver from `2.0.0` to `2.3.0`? 
`v2.0.1` is incorrect in this sentence.

Reply:
fixed.